custom_manifests/manifests/05-oci-eval-user-data.yml

```diff
@@ -0,0 +1,83 @@
+# ----- 05-oci-eval-user-data.yml -----
+
+# Generated by Butane; do not edit
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 00-master-oci-eval-user-data
+spec:
+  config:
+    ignition:
+      version: 3.4.0
+    storage:
+      files:
+        - contents:
+            compression: gzip
```

Comment:
does this have to be gzipped? makes it difficult to review/maintain the script.

Reply:
ideally, these manifests files should be build from the butane source files

Reply:
The script exists for review/maintenance in custom_manifests/butane/oci-eval-user-data-master.bu

Ideally we should keep all the butane files we use to generate MachineConfigs in custom_manifests/butane.